### PR TITLE
Update COM server and hosting samples to use new properties

### PR DESCRIPTION
**Pull request by elinor-fung**
`EnableDynamicLoading` and `EnableComHosting` properties were added to the .NET SDK in Preview7

### CODE REVIEW

File: core/extensions/COMServerDemo/ReadMe.md [hunk 1]
```diff
@@ -31,11 +31,11 @@ Program should output an estimated value of &#960;.
...
 
-1) Install .NET Core 3.0 Preview 5 or later.
+1) Install .NET Core 3.0 Preview 7 or later.
 
 1) Navigate to the root directory and run `dotnet.exe build /p:RegFree=True`.
 
-    - If the Registered COM demo was previously run, the project should be cleaned first - `dotnet.exe clean` 
+    - If the Registered COM demo was previously run, the project should be cleaned first - `dotnet.exe clean`
 
 1) Run the generated binary directly e.g. `COMClient\bin\Debug\netcoreapp3.0\COMClient.exe`.
```
Comment (mairaw) [inline core/extensions/COMServerDemo/ReadMe.md]:
```suggestion
1) Run the generated binary directly. For example, `COMClient\bin\Debug\netcoreapp3.0\COMClient.exe`.
```

**Comment by elinor-fung** [comment]
@mairaw I think waiting until Preview 7 ships would make sense.

**Comment by vitek-karas** [comment]
I added the small changes made in #1094 to the readme file.

**Comment by rpetrusha** [comment]
Since preview 7 has released, I'll merge this now, @elinor-fung.

**Pull request merged**